src/sst/elements/ariel/arielcore.cc

```diff
@@ -189,7 +198,7 @@ void ArielCore::commitWriteEvent(const uint64_t address,
         req->setVirtualAddress(virtAddress);
 
         if( writePayloads ) {
-            if(verbosity >= 16) {
+            //if(verbosity >= 16) {  //Verbosity is different than expected
```

Comment:
@shubhamn94 Can you explain this change?

Reply:
This change has been reverted in commit: 475de88 (latest commit after rebase). It was for debug purpose.